packages/orbit-components/src/Timeline/TimelineStep/index.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+import React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTheme from "../../defaultTheme";
+import Badge from "../../Badge";
+import Text from "../../Text";
+import Stack from "../../Stack";
+import CheckCircle from "../../icons/CheckCircle";
+import Circle from "../../icons/CircleEmpty";
+import AlertCircle from "../../icons/AlertCircle";
+import CloseCircle from "../../icons/CloseCircle";
+import { useStatuses, useStep } from "../TimelineContext";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+import type { Props } from "./index";
+
+const IconWrapperStyled = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: ${({ theme }) => theme.orbit.spaceLarge};
+  min-height: ${({ theme }) => theme.orbit.spaceLarge};
+  svg {
```

Comment:
Should not be necessary to use when the progress indicator will be rendered from the bottom edge of icon to the second one.